Sources/ElementaryHTMXWS/HTMLAttribute+HTMX.swift

```diff
@@ -0,0 +1,34 @@
+import Elementary
+import ElementaryHTMX
+
+public extension HTMLAttribute where Tag: HTMLTrait.Attributes.Global {
+    /// A namespace for the HTMX SSE-extension.
+    /// See the [htmx-sse reference](https://github.com/bigskysoftware/htmx-extensions/blob/main/src/sse/README.md) for more information.
+    enum ws {}
+}
+
+public extension HTMLAttribute.ws {
+    struct WSURLPrefix {
+        var value: String
+
+        public static func ws(_ url: String) -> Self {
+            .init(value: "ws:\(url)")
+        }
+
+        public static func wss(_ url: String) -> Self {
+            .init(value: "wss:\(url)")
+        }
+    }
+
+    static func connect(_ prefixedURL: WSURLPrefix) -> HTMLAttribute {
+        connect(prefixedURL.value)
+    }
```

Comment:
this feels a bit too much. isn't just string enough?

Reply:
Yes, probably true—couldn't decide on whether to add or not to add it.

Reply:
I think it should go - less API surface